conf/application.prod.conf

```diff
@@ -0,0 +1,26 @@
+# Secret key
+# Will be replaced in the future
+application.secret="WILL_BE_REPLACED"
+
+# The application languages
+application.langs="en"
+
+# Root logger:
+logger.root=ERROR
+
+# Logger used by the framework:
+logger.play=INFO
+
+# Logger provided to your application:
+logger.application=DEBUG
+
+# ReactiveMongo
+mongodb.uri = ${?MONGOLAB_URI}
+
+# Override does not work, any ideas are welcome
+mongo-async-driver {
+  akka {
+    loglevel = WARN

```

Comment:
Yes. In application.conf it is configured to be "loglevel = DEBUG" and it stays like that also for application.prod and application.test.conf. Mainly I've seen it while running tests and only by changing it in application.conf, then the debug messages are not show in the tests output.